client/src/main/scala/com.gu.contentapi.client/ContentApiClient.scala

```diff
@@ -44,9 +50,42 @@ trait ContentApiClient {
   /** Authentication and format parameters appended to each query */
   private def parameters = Map("api-key" -> apiKey, "format" -> "thrift")
 
+  val maxRetries: Int = 3 // max wait time ~1.5s
+
   /** Streamlines the handling of a valid CAPI response */
-  private def fetchResponse(contentApiQuery: ContentApiQuery)(implicit context: ExecutionContext): Future[Array[Byte]] =
-    get(url(contentApiQuery), headers).flatMap(HttpResponse.check)
+  private def fetchResponse(contentApiQuery: ContentApiQuery, attempt: Int = 1)(implicit context: ExecutionContext): Future[Array[Byte]] =
+    fetchResponseWithBackoff(contentApiQuery, attempt)(context).flatMap { _.flatMap { HttpResponse.check }}
+
+  private def fetchResponseWithBackoff(contentApiQuery: ContentApiQuery, attempt: Int)(implicit context: ExecutionContext): CancellableFuture[Future[HttpResponse]] = {
```

Comment:
The return type shows why this is wrong. The outer layer only makes sense if the inner future is created _inside_ of it.

Reply:
I don't think `attempt` is necessary here either.